src/nyc_trees/apps/users/templates/groups/settings.html

```diff
@@ -1,319 +1,313 @@
-                                    <a href="{% url 'add_event' group_slug=group_slug %}">+ New</a>
-                                </li>
-                            </ul>
-                            <div class="events">
-                                <div class="tab-content">
-                                    <div class="tab-pane active" id="current">
-                                        <a href="#" class="event block item">
-                                            <div class="row">
-                                                <div class="event-date col-xs-3">
-                                                    <div class="event-month">apr</div>
-                                                    <div class="event-day h4">25</div>
-                                                </div>
-                                                <div class="event-details col-xs-6">
-                                                    <h5 class="event-detail event-detail--title">tree like to party</h5>
-                                                    <div class="event-detail">4:30 pm</div>
-                                                    <div class="event-detail">450 may street</div>
-                                                </div>
-                                                <div class="event-action col-xs-3">
+                    </fieldset>
+                </form>
+
+            </div>
+
+            <div class="tab-pane" id="events">
+                <ul class="nav nav-tabs nav-tabs--filter">
+                    <li class="active">
+                        <a data-toggle="tab" href="#current">Current</a>
+                    </li>
+                    <li>
+                        <a data-toggle="tab" href="#past">Past</a>
+                    </li>
+                    <li>
+                        <a href="{% url 'add_event' group_slug=group_slug %}">+ New</a>
+                    </li>
+                </ul>
+                <div class="events">
+                    <div class="tab-content">
+                        <div class="tab-pane active" id="current">
+                            <a href="#" class="event block item">
+                                <div class="row">
+                                    <div class="event-date col-xs-3">
+                                        <div class="event-month">apr</div>
+                                        <div class="event-day h4">25</div>
+                                    </div>
+                                    <div class="event-details col-xs-6">
+                                        <h5 class="event-detail event-detail--title">tree like to party</h5>
+                                        <div class="event-detail">4:30 pm</div>
+                                        <div class="event-detail">450 may street</div>
+                                    </div>
+                                    <div class="event-action col-xs-3">
```

Comment:
This content was recently changed in https://github.com/azavea/nyc-trees/pull/262/, which replaced this prototype event list markup.  You should check with @steventlamb to prevent any merge conflicts.